R/plot_carp.R

```diff
@@ -82,160 +78,70 @@
 #' }
 plot.CARP <- function(x,
                       ...,
-                      type = c("dendrogram", "path", "dynamic_path", "js", "interactive"),
+                      type = c("dendrogram", "path", "js"),
```

Comment:
Type `js` seems a bit uninformative since `interactive = TRUE` seems to imply javascript. Would `heatmap` be better?

Reply:
Change `js` to `heatmap` would contribute to CMD check failure in `test_check("clustRviz", filter="carp_plot_static")` with `Message: 'arg' should be one of "dendrogram", "path", "heatmap"`

Reply:
Yeah - the tests will need to be changed as well. 

See https://github.com/DataSlingers/clustRviz/blob/452701543af8874edb52a5ca5907b5ac3ebeb6ef/tests/testthat/test_carp_plot_static.R#L64 for this line (I think)

Reply:
I think we also need to add test_carp_plot_interactive.R or something like that to test added interactive plots.

Reply:
Probably - but we should rethink these tests altogether at the end of the summer. Right now they don't do much other than just check they can run (they don't check that they are "right" in any sense) so I don't think it's such a big deal to have them in one file. Feel free to change the name if you think that makes sense.